code/game/objects/effects/effect_system.dm

```diff
@@ -228,6 +228,26 @@ steam.start() -- spawns the effect
 /////////////////////////////////////////////
 // Illumination
 /////////////////////////////////////////////
+/obj/effect/effect/light
```

Comment:
This file is for systems that spawn smaller effects, the light spot doesnt belong here.
May as well give it its own file in the folder,

Reply:
someday i refactor all effects, lets leave it till then